src/components/InvoiceTimer/InvoiceTimer.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react';
+import styled from 'styled-components';
+
+const TimeText = styled.p`
+	font-family: monospace;
+	font-size: 12px;
+	font-weight: 700;
+	line-height: 10px;
+	margin: 0;
+	display: grid;
+	text-align: left;
+	color: ${({ alert = false }) => (alert === true ? 'red' : '#000')};
+`;
+
+// Invoice Timer
+
+// Turn red if less than 1:00 left
+// align right so alignment doesn't change when time goes from 10:00 to 9:59
+// Only render if the invoice is active; not paid or expired
+type Props = {
+	secondsRemaining: ?number,
+};
+
+class InvoiceTimer extends React.PureComponent<Props> {
+	render() {
+		const { secondsRemaining } = this.props;
+		let isAlert = false;
+
+		let timeLeftMinutes = Math.floor(secondsRemaining / 60);
+		let remainderSeconds = secondsRemaining % 60;
+
```

Comment:
Can simplify the string manipulation logic and get rid of the if branching entirely with something like... 

```javascript
const formattedMinutes = `${timeLeftMinutes}`.padStart(2, '0')
const formattedSeconds = `${remainderSeconds}`.padStart(2, '0')
const formattedTime = `${formattedMinutes}:${formattedSeconds}`;
const isAlert = timeLeftMinutes < 1;

...

<TimeText alert={isAlert}>{formattedTime}</TimeText>
```

Reply:
a5a1960